src/NosCore.Parser/ImportFactory.cs

```diff
@@ -137,120 +139,120 @@ public void ImportAccounts()
                 Password = password
             };
 
-            if (_accountDao.FirstOrDefault(s => s.Name == acc1.Name) == null)
+            if (await _accountDao.FirstOrDefaultAsync(s => s.Name == acc1.Name).ConfigureAwait(false) == null)
             {
-                _accountDao.InsertOrUpdate(ref acc2);
+                acc2 = await _accountDao.TryInsertOrUpdateAsync(acc2).ConfigureAwait(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'acc2'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)